### PR TITLE
Support PVR Media

### PR DESCRIPTION
**Pull request by phunkyfish**
Used for testing of: https://github.com/xbmc/xbmc/pull/19395

### CODE REVIEW

File: pvr.demo/PVRDemoAddonSettings.xml [hunk 1]
```diff
@@ -1080,6 +1080,546 @@
       <radio>1</radio>
     </recording>
   </recordingsdeleted>
+  <media>
+    <!-- TV Media -->
+    <mediaTag>
+      <title>Demo TV Media Tag entry 1</title>
+      <episodetitle>Demo TV Media Tag 1 Episode Name</episodetitle>
+      <url>https://download.blender.org/demo/movies/BBB/bbb_sunflower_1080p_30fps_normal.mp4</url>
```
Comment (wsnipex) [inline pvr.demo/PVRDemoAddonSettings.xml]: did you want https://mirrors.kodi.tv/demo-files/BBB/ for this?

Comment (phunkyfish) [reply pvr.demo/PVRDemoAddonSettings.xml]: Yes definitely. I had forgotten to update that. Will do it shortly. Thanks!

**Comment by AlwinEsch** [comment]
Have changed the branch to the new Nexus.

**Comment by phunkyfish** [comment]
This was only a test PR and never meant to merged. Closing.

**Pull request closed without merging**